tests/test_webrtc_util.py

```diff
@@ -6,11 +6,11 @@
     _add_foundation_to_candidates,
     _get_media_direction,
     _update_direction_in_answer,
-    fix_mozilla_sdp_answer,
+    fix_sdp_answer,
 )
 
 
-def test_fix_mozilla_sdp_answer() -> None:
+def test_fix_sdp_answer() -> None:
```

Comment:
Can you split the new changes into a separate test? or make the test parametrized.